test/functional/testlib.bash

```diff
@@ -741,6 +814,18 @@ update_manifest() {
 	version | previous | filecount | timestamp | contentsize)
 		sudo sed -i "s/$key.*/$key:\\t$var/" "$manifest"
 		;;
+	minversion)
+		# Update minversion field which should only exist in MoM
+		sudo sed -i "s/$key.*/$key:\\t$var/" "$manifest"
```

Comment:
use:
sudo sed -i "s/^$key:.*/$key:\\t$var/" "$manifest"

Take a look at #586

Reply:
Nice! I'll fix that